tests/localosmosis/docker-compose.yml

```diff
@@ -11,10 +11,14 @@ services:
         RUNNER_IMAGE: alpine:3.16
         GO_VERSION: 1.18
     volumes:
+      - ../e2e/scripts/nativeDenomPool.json:/osmosis/nativeDenomPool.json
+      - ../e2e/scripts/nativeDenomThreeAssetPool.json:/osmosis/nativeDenomThreeAssetPool.json
       - ./scripts/setup.sh:/osmosis/setup.sh
       - ./.osmosisd/:/osmosis/.osmosisd/
     entrypoint:
       - /osmosis/setup.sh
+    command:
+      - $STATE
```

Comment:
Could we use environment variables instead of this?

In the other script, you can modify the `MONIKER` or `CHAIN_ID` via environment variables:

https://github.com/osmosis-labs/osmosis/pull/2795/files#diff-90a6bf7fea85b94101791de0f55d6ae2f059f900df53f24687e87ebeff4b5c7fR13-R15

and in the `docker-compose.yaml`:

https://github.com/osmosis-labs/osmosis/pull/2795/files#diff-19e0664081fa6ecc70118cd2c3b4204fcf23a2d63e2aba5a4d6fc294dfa342b0R20-R22

I think it's a little bit cleaner and consistent